pkgs/applications/blockchains/snarkvm/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "snarkVM";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "ProvableHQ";
+    repo = pname;
+    rev = "/refs/heads/mainnet";
```

Comment:
We do not allow packages whose source is a movable branch. Switch to a named tag or a commit SHA instead